crates/volta-core/src/inventory/mod.rs

```diff
@@ -1,15 +1,20 @@
 //! Provides types for working with Volta's _inventory_, the local repository
 //! of available tool versions.
 
+mod node;
+mod package;
+mod yarn;
+
 use std::collections::BTreeSet;
-use std::marker::PhantomData;
+use std::path::Path;
 
-use crate::tool::{Node, Package, Tool, Yarn};
+use failure::ResultExt;
 use lazycell::LazyCell;
+use regex::Regex;
 use semver::Version;
 use volta_fail::Fallible;
 
-pub(crate) mod serial;
+use crate::{error::ErrorDetails, fs::read_dir_eager, version::VersionSpec};
```

Comment:
Is this something that the formatter has started doing automatically? I see this style in a few places in this PR.

I like it visually, but I have avoided doing this in the past, because it makes rebasing harder. Separate PRs changing crate dependencies will be conflicting on this one line.

Reply:
Ah, that's interesting. I just started doing it myself—the formatter will leave whichever style you choose. If there's a preference by the rest of the team I'm happy to change it!